csharp/ql/test/query-tests/Security Features/CWE-079/XSSRazorPages/Generated/Template.g

```diff
@@ -0,0 +1,74 @@
+// A test file that mimics the output of compiling a `.cshtml` file
```

Comment:
How did you create this template?

Reply:
I took a generated file from the database from [this sample project](https://github.com/dsp-testing/csharp-f150), cut down to just the line containing the sink, and replaced anything dependent on the filepath with $PATHSLASH or $PATHUNDER.